eZ/Publish/API/Repository/Tests/UserServiceTest.php

```diff
@@ -3045,6 +3022,20 @@ public function testExpireUserToken($userToken)
         $userService->loadUserByToken($userToken);
     }
 
+    /**
+     * Test trying to load User by invalid Token.
+     *
+     * @covers \eZ\Publish\API\Repository\UserService::loadUserByToken
+     */
+    public function testLoadUserByTokenThrowsNotFoundException()
```

Comment:
Return type - missed this initially, my bad
```suggestion
    public function testLoadUserByTokenThrowsNotFoundException(): void
```